lib/vips/image.rb

```diff
@@ -35,7 +35,13 @@ module Vips
     rescue FFI::NotFoundError
     end
 
-    attach_function :vips_image_set, 
+    # vips_addalpha was added in libvips 8.6
+    if Vips::at_least_libvips?(8, 6)
```

Comment:
I was wondering if you want me to conditionally attach this function using `Vips::at_least_libvips?` (which actually makes the comment obsolete) or the same way you did above using `begin rescue`?

I vote for `at_least_libivips?` because it communicates more clearly why this doesn't get attached in some cases.

I'd change that in a different PR.

Reply:
I agree, at_least is clearer.